src/PantherDriver.php

```diff
@@ -164,6 +167,14 @@ public function visit($url)
         $this->client->get($this->prepareUrl($url));
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function getStatusCode()
+    {
+        return $this->client->getInternalResponse()->getStatusCode();
```

Comment:
Here we need to distinguish between the types of the client: PantherClient and HttpBrowserClient
The code would be similar to the following:
```PHP
public function getStatusCode()
{
    if ($this->client instanceof HttpBrowserClient) {
        return $this->client->getInternalResponse()->getStatusCode();
    }
    throw new UnsupportedDriverActionException('Status code is not available from "%s" with client of type "%s"', $this, \get_class($this->client));
}
```